stories/documentation/listings/index-table/index-table-actions-tooltipsRow.stories.ts

```diff
@@ -0,0 +1,96 @@
+import { provideHttpClient } from '@angular/common/http';
+import { provideAnimations } from '@angular/platform-browser/animations';
+import { LuUserPopoverDirective, provideLuUserPopover } from '@lucca-front/ng/popup-employee';
+import { LuTooltipModule } from '@lucca-front/ng/tooltip';
+import { LuUserDisplayModule, LuUserPictureComponent } from '@lucca-front/ng/user';
+import { applicationConfig, Meta, moduleMetadata, StoryFn } from '@storybook/angular';
+
+interface IndexTableActionsTooltipsRowStory {}
+
+export default {
+	title: 'Documentation/Listings/Index Table/Actions/Tooltips/Row',
+	argTypes: {},
+	decorators: [
+		moduleMetadata({
+			imports: [LuTooltipModule, LuUserPopoverDirective, LuUserDisplayModule, LuUserPictureComponent],
+		}),
+		applicationConfig({
+			providers: [provideAnimations(), provideLuUserPopover(), provideHttpClient()],
+		}),
+	],
+} as Meta;
+
+function getTemplate(args: IndexTableActionsTooltipsRowStory): string {
+	return `<table class="indexTable mod-layoutFixed">
+	<thead class="indexTable-head">
+		<tr class="indexTable-head-row">
+			<th class="indexTable-head-row-cell" scope="col">Action</th>
+			<th class="indexTable-head-row-cell" scope="col">Content</th>
+			<th class="indexTable-head-row-cell" scope="col">Content</th>
+			<th class="indexTable-head-row-cell" scope="col">Content</th>
+			<th class="indexTable-head-row-cell" scope="col" style="--components-indexTable-cell-fixed-width: 8">Content</th>
+			<th class="indexTable-head-row-cell mod-alignRight" scope="col" style="--components-indexTable-cell-fixed-width: 3"><span class="u-mask">Secondary action</span></th>
+		</tr>
+	</thead>
+	<tbody class="indexTable-body">
+		<!-- (click) with navigation on table row -->
+		<tr class="indexTable-body-row" (click)="message('Primary action')">
```

Comment:
In my opinion, `click` on `tr` should not be documented because `indexTable-body-row-cell-link` could then be seen as redundant and ignored / forgotten. 
It also add complexity by requiring some `$event.preventDefault(); $event.stopPropagation()`.

I much prefer documenting `(click)` on specifics `td`, when needed.